client/src/components/AuditAdmin/ActivityLog.tsx

```diff
@@ -73,12 +74,8 @@ const prettyAction = (activity: IActivity) => {
 const ActivityLog: React.FC = () => {
   const auth = useAuthDataContext()
   const user = auth && (auth.user as IAuditAdmin)
-  const organizations = useQuery<IOrganization[]>(
-    'orgs',
-    () => fetchApi(`/api/audit_admins/${user!.id}/organizations`),
-    { enabled: !!user }
-  )
-  if (!organizations.isSuccess) return null
```

Comment:
Factored out to use in `AuditAdminView` but later moved organization lookup to the backend. Still, seems like we prefer to organize query hooks outside components so I'm inclined to keep.